src/ansys_sphinx_theme/__init__.py

```diff
@@ -38,6 +38,22 @@ def get_version_match(semver):
     return ".".join([major, minor])
 
 
+def prepare_html_configration(app, pagename, templatename, theme_options, doctree):
```

Comment:
Do we really need to force the visibility of the switcher?

As of now, unless you enable it through your configuration, it will not be displayed. Also, forcing its appearance requires to impose a name to be displayed on it. Can we get the version number from the `app` parameter?